src/System.IO.Ports/tests/Legacy/Support/TCSupport.cs

```diff
@@ -605,5 +622,71 @@ public static void VerifyArray<T>(T[] expectedArray, T[] actualArray, int index,
                 }
             }
         }
+
+        /// <summary>
+        /// Set both ports to 115200 baud to speed test performance
+        /// </summary>
+        public static void SetHighSpeed(SerialPort com1, SerialPort com2)
+        {
+            if (com1 != null)
+            {
+                com1.BaudRate = 115200;
+            }
+            if (com2 != null && com2 != com1)
+            {
+                com2.BaudRate = 115200;
+            }
+        }
+
+
+        /// <summary>
+        /// Wait for the write data to be written into a blocked (by adverse flow control) port
+        /// </summary>
+        public static void WaitForWriteBufferToLoad(SerialPort com, int bufferLength)
+        {
+            Stopwatch sw = Stopwatch.StartNew();
+            while (com.BytesToWrite + HardwareTransmitBufferSize < bufferLength)
+            {
+                System.Threading.Thread.Sleep(50);
+                if (sw.ElapsedMilliseconds > 3000)
```

Comment:
Yes, good point - I don't know why I didn't do that there - if you look a few lines further down in that file I've done exactly what you said - I'll tidy that up.